framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchKieServerPersistentScenarioImpl.java

```diff
@@ -147,4 +151,42 @@ private void storeProjectInfoToPersistentVolume(Deployment deployment, String pe
     public SsoDeployment getSsoDeployment() {
         return ssoDeployment;
     }
+
+    @Override
+    public void changeUsernameAndPassword(String username, String password) {
+        if(getDeployments().stream().allMatch(Deployment::isReady)) {
+            deploySecretAppUser(username,password);
+            logger.info("Restart the environment to update Workbench deployment.");
+            getDeployments().parallelStream().forEach(this::scaleToZeroAndBackToReplicas); // if parallel stream make mess because of common fork-join pool use normal stream and adjust scaling (scale all deployments to zero at the same time)
```

Comment:
I think it's fine to use parallelStream, my only concern is that we'll be restarting all the deployment units (including databases in persistence scenarios. 
Maybe, as an alternative approach, we could make the scaleToZeroAndBackToReplicas (or renamed to just "restart") into the OpenshiftDeployment interface and not to do anything in the database deployments. 
What do you think?